doc/STYLE_GUIDE.md

```diff
@@ -1,28 +1,29 @@
 # Style Guide
 
 * Documents are written in markdown files.
-* Those files should be written in **`lowercase-with-dashes.md`.**
+* Those files should be written in **`lowercase-with-dashes.md`**.
   * Underscores in filenames are allowed only when they are present in the
-    topic the document will describe (e.g., `child_process`.)
+    topic the document will describe (e.g., `child_process`).
   * Filenames should be **lowercase**.
   * Some files, such as top-level markdown files, are exceptions.
   * Older files may use the `.markdown` extension. These may be ported to `.md`
     at will. **Prefer `.md` for all new documents.**
 * Documents should be word-wrapped at 80 characters.
 * The formatting described in `.editorconfig` is preferred.
-  * A [plugin][] is available for some editors to automatically apply these rules.
+  * A [plugin][] is available for some editors to automatically apply these
+    rules.
 * Mechanical issues, like spelling and grammar, should be identified by tools,
   insofar as is possible. If not caught by a tool, they should be pointed out by
   human reviewers.
 * American English spelling is preferred. "Capitalize" vs. "Capitalise",
   "color" vs. "colour", etc.
 * Though controversial, the [Oxford comma][] is preferred for clarity's sake.
 * Generally avoid personal pronouns in reference documentation ("I", "you",
-  "we".)
+  "we").
   * Pronouns are acceptable in more colloquial documentation, like guides.
   * Use **gender-neutral pronouns** and **mass nouns**. Non-comprehensive
     examples:
-    * **OK**: "they", "their", "them", "folks", "people", "developers", "cats"
+    * **OK**: "they", "their", "them", "folks", "people", "developers", "cats".
```

Comment:
Nit: This line and the following line probably shouldn't end with periods because they are not sentences. They are just lists. I'm OK with using a period anyway for consistency or simplicity, but I do have a slight preference for leaving the period off in this case.